examples/invoker/client.go

```diff
@@ -88,66 +94,68 @@ func main() {
 		defer shutdown()
 	}
 
-	realRPS := runBenchmark(urls, *runDuration, *rps)
+	realRPS := runBenchmark(endpoints, *runDuration, *rps)
 
 	writeLatencies(realRPS, *latencyOutputFile)
 }
 
-func readLines(path string) ([]string, error) {
-	file, err := os.Open(path)
+func readEndpoints(path string) (endpoints []Endpoint, _ error) {
+	data, err := ioutil.ReadFile(path)
 	if err != nil {
 		return nil, err
 	}
-	defer file.Close()
-
-	var lines []string
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		lines = append(lines, scanner.Text())
+	if err := json.Unmarshal(data, &endpoints); err != nil {
+		return nil, err
 	}
-	return lines, scanner.Err()
+	return
 }
 
-func runBenchmark(urls []string, runDuration, targetRPS int) (realRPS float64) {
+func runBenchmark(endpoints []Endpoint, runDuration, targetRPS int) (realRPS float64) {
 	timeout := time.After(time.Duration(runDuration) * time.Second)
 	tick := time.Tick(time.Duration(1000/targetRPS) * time.Millisecond)
 
 	var issued int
 	start := time.Now()
 
+	Start(TimeseriesDBAddr, endpoints)
```

Comment:
Keep initialization off the measurement phase
```suggestion
	Start(TimeseriesDBAddr, endpoints)

	start := time.Now()
```